### PR TITLE
docs: updates readme

### PR DESCRIPTION
**Pull request by TuringLovesDeathMetal**
Updates readme with new logo and adds Lightdash Values.

### CODE REVIEW

**Comment by TuringLovesDeathMetal** [comment]
@owlas I know you were saying that the features list was quite old...but I couldn't think of "jazzier" features to put in there 🤔 any suggestions welcome!

**Comment by TuringLovesDeathMetal** [comment]
@hamzahc1 I added the new logo as a sort of "banner", but it doesn't look great 🥴 

The other option is to use the logo without the background, but I'm not sure how the all-black version would appear on dark theme GitHub 🤔 

Let me know what you think!

**Review by owlas — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <h1 align="center">
-        <a href="https://www.lightdash.com">⚡️ Lightdash </a>
+        <a target="_blank" href="https://www.lightdash.com"><img align="center" style="max-width:300px;" src="/static/screenshots/lightdash_logo_banner.png"> </a>
```
Comment (hamzahc1) [inline README.md:2]: @TuringLovesDeathMetal We can either put in a light mode logo like below: 

Github Inverted Logo (1)

or stick to one with a coloured background that works on both: 
OpenGraph Image (3)

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 </h1>
 
-<p align="center">The open source Looker alternative</p>
+<p align="center">Open source BI for your whole team.</p>
```
Comment (hamzahc1) [inline README.md:5]: For the readme I preferred the open-source looker alternative, it feels more "this is what it does". Wdyt?

File: README.md [hunk 3]
```diff
@@ -28,10 +28,11 @@
...
 </div>
 
-Lightdash is a BI tool that is fully integrated with your dbt project, allowing you to define metrics alongside your
-data models.
+Connect Lightdash to your dbt project, add metrics directly in your data transformation layer, then create and share your insights with your team.
 
-## Features
+Our mission is simple: we want to enable everybody in your company to answer their own questions using data.
+
+## Features:
```
Comment (hamzahc1) [inline README.md:35]: Can we update the feature list too? Some example features we could include: dashboards & saved charts, table calculations, lineage, export functionality.

**Pull request merged**